index.html

```diff
@@ -10,15 +9,15 @@
   </head>
 
   <body>
-  <div id="home-page">  
+  <div id="home-page"> 
     <header class="header">
         <a href="#" class="logo">My Logo</a>
         <nav id="nav-hamburger">
-            <img src="/images/nav.png">        
+        <img src="/images/nav.png">        
         </nav>
     </header>
     <section id="main">
-        <h1 class="main-heading">I'm Celyne.<br> Glad to see you!</h1>
+        <h1 class="main-heading">I'm Evelyne Celyne<br> Glad to see you!</h1>
         <p class="main-paragraph">I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
         <label class="connect">LET'S CONNECT</label>
```

Comment:
- [ ] Please ensure the "LET'S CONNECT" is either in a link tag like the `a` tag or in a `button` element.